README.md

```diff
@@ -16,7 +16,6 @@ Various subdirectories within the repo contain a kustomize target (base or overl
 📦  gcp               ⇲     
                       ⎹→🗳  gcp/cert-manager/overlays/gcp(🎯)
```

Comment:
The README.md directory list can be regenerated by running hack/gen-tree.sh (in #50)

I've added it here for convenience

```
.
└── manifests
    ├── admission-webhook
    │   ├── podpreset_crd
    │   │   └── 🎯base
    │   └── webhook
    │       └── 🎯base
    ├── application
    │   └── 🎯base
    ├── argo
    │   └── 🎯base
    ├── common
    │   ├── ambassador
    │   │   └── 🎯base
    │   ├── basic-auth
    │   │   └── 🎯base
    │   ├── centraldashboard
    │   │   └── 🎯base
    │   └── spartakus
    │       └── 🎯base
    ├── gcp
    │   ├── cert-manager
    │   │   └── overlays
    │   │       └── 🎯gcp
    │   ├── cloud-endpoints
    │   │   └── overlays
    │   │       └── 🎯gcp
    │   ├── gpu-driver
    │   │   └── overlays
    │   │       └── 🎯gcp
    │   └── iap-ingress
    │       └── overlays
    │           └── 🎯gcp
    ├── jupyter
    │   ├── jupyter
    │   │   ├── overlays
    │   │   │   └── 🎯minikube
    │   │   └── 🎯base
    │   ├── jupyter-web-app
    │   │   └── 🎯base
    │   └── notebook-controller
    │       └── 🎯base
    ├── katib
    │   └── 🎯base
    ├── kubebench
    │   └── 🎯base
    ├── metacontroller
    │   └── 🎯base
    ├── modeldb
    │   └── 🎯base
    ├── pipeline
    │   ├── api-service
    │   │   └── 🎯base
    │   ├── minio
    │   │   └── 🎯base
    │   ├── mysql
    │   │   └── 🎯base
    │   ├── persistent-agent
    │   │   └── 🎯base
    │   ├── pipelines-runner
    │   │   └── 🎯base
    │   ├── pipelines-ui
    │   │   └── 🎯base
    │   ├── pipelines-viewer
    │   │   └── 🎯base
    │   └── scheduledworkflow
    │       └── 🎯base
    ├── profiles
    │   └── 🎯base
    ├── pytorch-job
    │   └── pytorch-operator
    │       └── 🎯base
    ├── tensorboard
    │   └── 🎯base
    └── tf-training
        └── tf-job-operator
            └── 🎯base
```

Reply:
👍